Gemfile

```diff
@@ -10,6 +10,9 @@ ruby RUBY_VERSION
 # This is the default theme for new Jekyll sites. You may change this to anything you like.
 gem "minima", "~> 2.0"
 
+# Use nokogiri >= 1.10.4 (contains fix of CVE-2019-5477)
+gem "nokogiri", ">= 1.10.4"
```

Comment:
Is this needed? Just running bundle update seems to update the Gemfile.lock file with the new nokogiri already.